### ISSUE TITLE
parallel root text linking to a 404 page in Safari

### ISSUE THREAD

**Issue by delightedbydhamma** [suttacentral/suttacentral#3221]
#### Description:
While trying to find an easy way to click from bhikkhuni to bhikkhu rules I clicked on the root text under parallels and was taken to https://suttacentral.net/pli-tv-bi-vb-np21/null/?lang=en which throws a error 404 in Safari (Version 17.4.1 (19618.1.15.11.14) desktop.

In Chrome the URL changes to https://suttacentral.net/pli-tv-bi-vb-np13/null/ but the page doesn't change.

#### Steps to reproduce:
navigate to list of rules (ie [bhikkhuni > nissaggiya pacittiyas](https://suttacentral.net/pli-tv-bi-vb-np?view=dense&lang=en))
click on parallel for rule arrow (ie out of season robe cloth NP15)
click on the box that says 'pli-tv-bi-vb-np15'
ERROR

#### Expected behaviour:
 Just trying to find my way to the bhikkhu rule!

**Comment by thesunshade** [comment]
Are you talking about the box I'm pointing to in this screenshot?

![image](https://github.com/user-attachments/assets/da920609-7e60-4f13-95ef-d0727086f44f)

Because it's not clickable at all for me.

Strangely, it has an `<a>` tag, but it's an empty href.

![image](https://github.com/user-attachments/assets/4c931ab2-a07d-475e-8fe4-09609d4f878e)

**Comment by delightedbydhamma** [comment]
No. That's not a bug, that's a feature request :)

I'm talking about the one I've circled in green (recycling screenshots)
<img width="794" alt="1Screenshot 2024-07-18 at 2 57 19 PM" src="https://github.com/user-attachments/assets/1ff5a539-b0fd-4a1a-a38e-51545700dcb5">

**Comment by thesunshade** [comment]
Oh, sorry. I almost never use the parallels. And the list is so long I missed the fact that the box had text in it. I don't think I even realized they were clickable.

![image](https://github.com/user-attachments/assets/0a69057a-4e28-4418-bf62-e941e1ca519c)

> No. That's not a bug, that's a feature request :)

You mean "Just trying to find my way to the bhikkhu rule!" is your feature request?

I'm confused.

Shouldn't the expected behaviour be that it's not clickable?

Are you saying that that box should be linking to the Bhikkhu Vinaya manuscript? It's a strange situation where we are giving parallels to a non-existant manuscript.

Apologies, I also don't spend much time digging in the Vinaya so I'm not as familiar with the structure of the texts.

**Comment by delightedbydhamma** [comment]
Sorry. Ignore 'Just trying to find my way...' 

I was confused too. I wonder if it would be better if the text was vertical aligned top?

I don't know what the expected behaviour should be. 
The text in the box is the manuscript which we are  already looking at.

## expected behaviour 
OK, I did some more digging and it is returning 'null' where it should be returning the pali manuscript.
Checking DN2 and if you click that box it goes straight to the pali text of DN2

--
As an aside and maybe another related ticket.
On DN2 the parallels which exist on SC are also clickable, but until you hover over them, there is no clue which one exists. 
By this logic the parallels for the bhikkhuni rules (pali-bhikkhu) should be clickable.

**Comment by sujato** [comment]
Okay, thanks for that, there are a few issues here. 

The bigger one is that the schools of the Vinaya are not showing, this is a regression and I'll list it separately.

But the specific problem here is dealing with Bhikkhuni Vinaya rules that only exist in theory, i.e. those assumed to be held in common with the bhikkhu rules. There's no text for these, which is what causes the `null`. 

Generally, when there is no text associated with a parallels entry, we assign `class='disabled'`. But it seems this only works for the parallels, not for the primary text. 

Currently we have:

```
<a class="root-link" href="/pli-tv-bi-vb-np16/null/">
<div class="parallels-root-id root" title="SuttaCentral ID">
Pli Tv Bi Vb NP 16
</div>
<md-ripple aria-hidden="true"></md-ripple>
</a>
```

It should be:

```
<a class="root-link disabled">
<div class="parallels-root-id root" title="SuttaCentral ID">
Pli Tv Bi Vb NP 16
</div>
</a>
```

In other words:

- [x] add class `disabled`
- [x] remove `href`
- [x] remove ripple

Now, for the parallels, we can improve a litlle. Currently we have:

```
<a href="" class="disabled">
        <div class="parallel-item">
          <div class="parallel-item-main-info-container">
            <div class="parallel-item-title" title="SuttaCentral ID">Bhikkhu Parivāra Nissaggiya Pācittiya 6</div>
            <div class="parallel-item-nerdy-row">
            </div>
          </div>
        </div>
      </a>
```

- [x] remove `href="" `.

Also, the styling on these should communicate their inactive state more clearly, I'll have a think about that.

**Comment by ihongda** [comment]
@delightedbydhamma  I have fixed this problem and pushed to `Staging`.

https://staging.suttacentral.net/pli-tv-bi-vb-np15?view=normal&lang=en

**Comment by delightedbydhamma** [comment]
Great! It behaves as Bhante Sujato describe.
